modules/backend/formwidgets/Repeater.php

```diff
@@ -235,20 +233,15 @@ protected function processItems()
                 }
             }
         }
-
-        if (is_array($currentValue) && count($currentValue)) {
-            foreach ($currentValue as $value) {
-                $groupMap[] = array_get($value, '_group');
-            }
-        }
-
-        if (!count($groupMap)) {
+        
+        if (!is_array($currentValue)) {
             return;
         }
-
-        foreach ($groupMap as $index => $groupCode) {
-            $this->makeItemFormWidget($index, $groupCode);
-        }
+       
+        collect($currentValue)->each(function ($value, $index) {
+            $this->makeItemFormWidget($index, array_get($value, '_group', false));
```

Comment:
@datune Sorry, one last thing. The default value if `_group` isn't available should be `null`, not `false`.
```suggestion
            $this->makeItemFormWidget($index, array_get($value, '_group', null));
```